puphpet/config.yaml.template

```diff
@@ -79,25 +100,31 @@ apache:
     settings:
         user: www-data
         group: www-data
-        default_vhost: true
+        default_vhost: false
         manage_user: false
         manage_group: false
         sendfile: 0
     modules:
+        - cache
+        - deflate
+        - expires
+        - headers
+        - proxy
         - rewrite
+        - ssl
     vhosts:
-        snwhpze4b6vn:
-            servername: awesome.dev
+        vagrant_dev:
+            servername: vagrant.dev
             serveraliases:
-                - www.awesome.dev
-            docroot: /var/www/awesome
+                - www.vagrant.dev
+            docroot: /var/www/webroot
             port: '80'
             setenv:
-                - 'APP_ENV dev'
+                - 'APP_ENV vagrant'
             directories:
-                duosfqifgtqo:
+                var_www:
                     provider: directory
-                    path: /var/www/awesome
+                    path: /var/www
```

Comment:
I haven't tested Apache so I don't know if it needs the to be set to `/var/www/webroot` but wouldn't that be best practice anyway?